### PR TITLE
[Jetsnack] Implement "see more" for Details if its too long

### PR DESCRIPTION
**Pull request by angeles-bilbao6**
Implemented missing feature SEE MORE

Before
image

After
image
image

### CODE REVIEW

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt [hunk 1]
```diff
@@ -165,13 +167,32 @@ private fun Body(
                         modifier = HzPadding
                     )
                     Spacer(Modifier.height(16.dp))
+                    val seeMore = remember { mutableStateOf(true) }
```
Comment (nickbutcher) [inline Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt:170]: Use delegate

```suggestion
                    val seeMore by remember { mutableStateOf(true) }
```

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt [hunk 2]
```diff
@@ -165,13 +167,32 @@ private fun Body(
...
                     Text(
                         text = stringResource(R.string.detail_placeholder),
                         style = MaterialTheme.typography.body1,
                         color = JetsnackTheme.colors.textHelp,
+                        maxLines = if (seeMore.value) 5 else Int.MAX_VALUE,
+                        overflow = TextOverflow.Ellipsis,
                         modifier = HzPadding
                     )
-
+                    val textButton = if (seeMore.value)
```
Comment (nickbutcher) [inline Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt:179]: More meaningful name please

Comment (nickbutcher) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt:179]: Also can we have curly braces around the clauses please (or inline the `if` within the `stringResource`?)

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt (hunk 1 above)
Comment (angeles-bilbao6) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt:170]: If I replace I get an error
image

Comment (nickbutcher) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt:170]: You'll also need to add 
```
import androidx.compose.runtime.getValue
import androidx.compose.runtime.setValue
```

**Review by nickbutcher — APPROVED** [review]

**Pull request merged**